### PR TITLE
Feature/ask

### PR DESCRIPTION
**Pull request by binobi**

### CODE REVIEW

**Comment by binobi** [comment]
Note, this doesn't support sending example `yes` and `no` images in the request.

File: avabot/commands/ask.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        image_urls = self.kwargs['<urls>']
+        question = self.kwargs['--question']
+
+        logging.info(f'posting to /v2/ask - urls={image_urls}')
+        images = [{'url': image} for image in image_urls]
+        try:
+            return self.ii_client.ask(images, question, custom_id='ava-slackbot-' + str(uuid.uuid4()))
+        except ApiRequestError as e:
+            logging.info(f'failed to POST /v2/ask - urls={image_urls}, error={e}')
+            raise e
```
Comment (davidvuong) [inline avabot/commands/ask.py]: You don't need to change this but as Seb mentioned in one my own PRs, you can use `raise` to re-raise the current exception.

Comment (binobi) [reply avabot/commands/ask.py]: :wow

File: avabot/vendor/slack.py [hunk 2]
```diff
@@ -41,6 +41,10 @@ def _parse_message(self, message):
         user = message.get('user')
         type_ = message.get('type')
         text = message.get('text')
+
+        # replace slack's unicode quotes with the normal ones so it can be parsed correcty
+        text = str(text).replace(u'\u201c', '"').replace(u'\u201d', '"')
```
Comment (davidvuong) [inline avabot/vendor/slack.py]: Not sure if this desired behaviour but `str(None)` gives `'None'`. `.get` on a key that doesn't exist returns `None` above.

Comment (binobi) [reply avabot/vendor/slack.py]: Good point. I'll move this after the `should_respond` check, just to be safe. That filters out msgs with `text = None`.

**Pull request merged**